ui/src/Trade/ExchangeInterface.jsx

```diff
@@ -3,30 +3,41 @@ import { parseAsAmount, stringifyValue } from '@agoric/ui-components';
 import { useStore } from '../store/store.js';
 import { buyOffer, makeGenericOnStatusUpdate, sellOffer } from '../utils/makeOrder.js';
 
-export default function ExchangeInterface() {
+export function ExchangeInterface() {
   const [inputValue, setInputValue] = useState('');
   const [outputValue, setOutputValue] = useState('');
   const [assetLabel, setAssetLabel] = useState('Asset:');
   const [priceLabel, setPriceLabel] = useState('Price:');
-  const [assetBrandName, setAssetBrandName] = useState('BLD');
-  const [priceBrandName, setPriceBrandName] = useState('IST');
+  const globalAssetBrandName = useStore((state) => state.assetBrandName);
+  const globalPriceBrandName = useStore((state) => state.priceBrandName);
+  const [assetBrandName, setAssetBrandName] = useState(`${globalAssetBrandName}`);
+  const [priceBrandName, setPriceBrandName] = useState(`${globalPriceBrandName}`);
   const [isOpen, setIsOpen] = useState(false);
   const wallet = useStore((state) => state.wallet);
   const notifyUser = useStore((state) => state.notifyUser);
   const assetBrand = useStore((state) => state.assetBrand);
-  const priceBrand = useStore((state) => state.assetBrand);
+  const priceBrand = useStore((state) => state.priceBrand);
   const getDisplayInfo = useStore((state) => state.getDisplayInfo);
   const isBuyOrder = assetLabel === 'Price:';
 
   const vbankAssets = useStore((state) => state.vbankAssets);
-  const { onStatusChange } = makeGenericOnStatusUpdate(notifyUser);
+  const resetInputFields = () => {
+    setInputValue('');
+    setOutputValue('');
+  };
+  const { onStatusChange } = makeGenericOnStatusUpdate(notifyUser, resetInputFields);
 
   useEffect(() => {
     if (vbankAssets.length > 0) {
       console.log('vbankAssets in Exchange:', vbankAssets);
     }
   }, [vbankAssets]);
 
```

Comment:
What is the reason for this? If it's to make the component rerender when `vbankAssets` changes `useStore((state) => state.vbankAssets);` is sufficient for that